### DISCUSSION TITLE
Flow through Swipes

### DISCUSSION THREAD

**Discussion by renieness** [SwipesForScience/SwipesForScience#123] [Show and tell]
Hello, I've been rethinking the flow for new users for an Educational vs Community science game. 

![Community Game - New User](https://user-images.githubusercontent.com/39166059/111926279-53601a00-8b00-11eb-8317-8c5c28798dc7.png)

![Educational Game - New User](https://user-images.githubusercontent.com/39166059/111926285-5824ce00-8b00-11eb-8fea-f5093ea101b2.png)

The main difference between an Educational game and a Community game is that the educational game won't have a leaderboard (I think, please correct me if I'm wrong @jyeatman).

I'm still trying to figure out where returning educational players will land when they bookmark a game, or are prompted by email (?) to play. (Is this even a consideration?)

**Comment by jyeatman** [comment]
Yes, I think this makes sense. Basically we are allowing a configuration for whether the userboard and chat show up right?
